### PR TITLE
store check result

### PR DESCRIPTION
**Pull request by pnuckowski**
* stored in cassandra
* couchbase removed
* tests

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -38,6 +34,23 @@ Information
...
+------------
+
+Alamo-alerter use Cassandra database to store check results.
+
+To create proper table in Cassandra execute following CQL command in CQL interactive terminal (`cqlsh`)::
+
+    create table check_results(
+        check_id int,
+        insertion_time timestamp,
+        result boolean,
```
Comment (pnuckowski) [inline README.rst]: Maybe not in this PR but renaming `result` to `is_failed` could be more understandable. Right? :)

Comment (deejay1) [inline README.rst]: no idea, I think "is_failed" would be even worse. Also we'd need to check the clustering key with Marek...

File: alamo_alerter/alerter.py [hunk 2]
```diff
@@ -6,21 +6,26 @@
 from collections import deque
 
 from requests import Session
-from requests.exceptions import HTTPError
+from requests.exceptions import HTTPError, ConnectionError
 
-from alamo_alerter.cache import CouchBaseCache
+from alamo_alerter.backend.cassandra import CassandraDriver
```
Comment (deejay1) [inline alamo_alerter/alerter.py]: wasn't the whole idea to have them swappable? ;)

File: alamo_alerter/backend/cassandra.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+from cassandra.auth import PlainTextAuthProvider
+from cassandra.cluster import Cluster
+
+
+class CassandraDriver(object):
+    # cassandra `cluster` instance
+    cluster = None
```
Comment (deejay1) [inline alamo_alerter/backend/cassandra.py]: I'd "mark" that as private

File: tests/test_cassandra_backend.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+from unittest.mock import call
+from datetime import datetime
+
+from mock import patch
+from mock.mock import MagicMock
+
+from alamo_alerter.backend.cassandra import CassandraDriver
+
+
+class CouchBaseTest(unittest.TestCase):
```
Comment (deejay1) [inline tests/test_cassandra_backend.py]: CassandraDriverTest ? ;)

File: tests/test_cassandra_backend.py [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    @classmethod
+    def setUpClass(cls):
+        cls.check_id = 999
+
+    @patch('alamo_alerter.backend.cassandra.Cluster')
```
Comment (deejay1) [inline tests/test_cassandra_backend.py]: I wonder if we really have to mock this, as travis provides a cassandra instance

**Comment by deejay1** [comment]
:+1:

**Comment by ch40s3k** [comment]
:+1:

**Pull request merged**